vrk-kpa/suomidigi
# SUOPA-658 Events

To test: 
- `make fresh`
- Go to https://suomidigi.docker.sh/node/add/event and create few events with at least
   - Title, Location, Start date and description
   - Make sure some of the events are in the future and some in the past
- Go to https://suomidigi.docker.sh/node/add/landing_page 
   - Name it "Tapahtumat"
   - Add a slogan for the page
   - Create View paragraph and select Events for the view and Upcoming events for the Display
   - Duplicate the view (from three dots-menu) and select Past events for the duplicated view, like so:![image](https://user-images.githubusercontent.com/1712902/69068354-44a9b480-0a2d-11ea-9318-33a77cf11319.png)
- Save the node and check that it lists your events in correct fashion.
- Also click one of the events and check that there's a button for returning to the list stating "Katso kaikki tapahtumat"
    - This button will most likely point to wrong node, but it can be set to correct one from configuration [ /conf/cmi/suopa_events.settings.yml ]
